src/components/Search.tsx

```diff
@@ -110,7 +110,6 @@ export function Search({
         createPortal(
           <DocSearchModal
             {...options}
-            initialScrollY={window.scrollY}
```

Comment:
Awesome stuff.

is this the actual fix? Can you explain what the problem was and how this fixes it?

Reply:
Hi @rickhanlonii 👋

Thanks so much! Yep, line 113 is the actual fix. I didn't take a look into the internals of the library, but it looks like the bug might actually be in the [Autocomplete library](https://autocomplete-experimental.netlify.app/docs/docsearchmodal/).  It looks like their DocSearchModal component has a bug in the modal design, which is causing the screen to freeze once the device width is less than about 755px.

https://github.com/reactjs/react.dev/assets/57572988/305696e2-52b5-4b7b-a362-cb66140d0ebe

Reply:
Hi @rickhanlonii 👋

Thanks so much! Yep, line 113 is the actual fix. I didn't take a look into the internals of the library, but it looks like the bug might actually be in the [Autocomplete library](https://autocomplete-experimental.netlify.app/docs/docsearchmodal/).  It looks like their DocSearchModal component has a bug in the modal design, which is causing the screen to freeze once the device width is less than about 755px.

https://github.com/reactjs/react.dev/assets/57572988/305696e2-52b5-4b7b-a362-cb66140d0ebe